rest/data/commit_queue.go

```diff
@@ -264,7 +266,7 @@ func (pc *DBCommitQueueConnector) IsItemOnCommitQueue(id, item string) (bool, er
 	return false, nil
 }
 
-func (pc *DBCommitQueueConnector) CommitQueueClearAll() (int, error) {
+func CommitQueueClearAll() (int, error) {
```

Comment:
Can we get rid of this function and just call commitqueue.ClearAllCommitQueues directly? (We can probably get rid of the tests too since ClearAllCommitQueues has its own tests)